model.py

```diff
@@ -4599,6 +4599,21 @@ def to_search_documents(cls, works):
             )
         ).alias('works_alias')
 
+        # This subquery gets Collection IDs.
+        collections = select(
+            [LicensePool.collection_id]
+        ).where(
+            LicensePool.work_id==literal_column(works_alias.name + '.' + works_alias.c.work_id.name)
+        ).alias("collections_subquery")
+
+        # Create a json array from the set of Collections.
+        collections_json = select(
+            [func.array_to_json(
+                    func.array_agg(
+                        func.row_to_json(
+                            literal_column(collections.name)
```

Comment:
This should be collections.id, not collections.name.

Reply:
Or not, I think this is the name of the subquery and it works because it only has one column.

Reply:
I originally thought the same, but this isn't pulling a field out of the 'collections_subquery' alias, it's turning the alias itself (with _all_ of its fields) into a JSON dictionary. `collections.name` here is 'collections_subquery', the name of the subquery.

You'll see below this code that `contributors_json` does something similar. It generates SQL that looks kind of like this toy example:

```
select row_to_json(subquery) from (select sort_name, family_name from contributors limit 1) as subquery;
```

The results:

```
{"sort_name":"Sen, Chaitali","family_name":null}
```

In the toy example, row_to_json() is called on "subquery", not "subquery.sort_name" or "subquery.family_name"